vault-tool/src/Network/VaultTool/Transit.hs

```diff
@@ -69,34 +75,20 @@ decodeBase64 = B64.decode . getBase64
 
 encryptBase64 :: VaultConnection Authenticated -> VaultMountedPath -> KeyName -> Base64 -> IO CipherText
 encryptBase64 conn path key =
-    parseResponse <=< runVaultRequestAuthenticated conn . newPostRequest (mkUri path key) . mkBody
+    maybe (throwIO . userError $ "Unexpected response from vault transit encrypt") (pure . getCyphertext) <=< runVaultRequestAuthenticated conn . newPostRequest (mkUri path key) . mkBody
   where
-    mkUri (VaultMountedPath p) k = p <> "/" <> k
+    mkUri (VaultMountedPath p) k = T.intercalate "/" [p, "encrypt", k]
```

Comment:
out of curiosity, is the `encrypt` and `decrypt` portion of these paths just a standard we are enforcing or is that how vaults up the key?

Reply:
[Seems like it's part of how Vault exposes access to the keys](https://developer.hashicorp.com/vault/api-docs/secret/transit#encrypt-data)